### DISCUSSION TITLE
Can't upload images to Editor Notes

### DISCUSSION THREAD

**Discussion by tubbecool** [leekelleher/umbraco-contentment#292] [Discussion]
Hi, I'm having issues uploading media to an Umbraco.Community.Contentment.EditorNotes.

I'm picking the media under Message, but nothing appears:

![image](https://user-images.githubusercontent.com/47563232/210232885-7c7c3b01-d766-4cae-8f0e-30ff82bf5272.png)

It just remains empty. 

Is this a known error? 
I'm on Umbraco 11 and Contentment version 4.4.0. (It worked on Umbraco 10)

**Comment by leekelleher** [comment]
@tubbecool Thanks for reporting this. My first day back, post-Xmas/NY, I'll take a look on Umbraco 11. 👍

> Note for self, Umbraco 11 did upgrade TinyMCE to v6 - could be related? 🤔 
> https://github.com/umbraco/Umbraco-CMS/pull/13013

**Reply by leekelleher** [reply]
Hi @tubbecool, I've tried this out on Umbraco 11 (with latest Contentment version) and it's working fine for me.
In the Editor Notes' Message field, I can pick from existing media images and upload new images, and they'll appear in the rich text editor.

Do you know if you get any JavaScript errors in your browser's dev tools console? <kbd>F12</kbd>

**Reply by tubbecool** [reply]
Sorry for late reply. I've been on vacation.

This is the console error:

![image](https://user-images.githubusercontent.com/47563232/215732665-e0fc3ff3-a346-4a72-bc2e-26f76c058383.png)

I'm using Azure blob storage for media files.
